test_conformance/subgroups/CMakeLists.txt

```diff
@@ -7,6 +7,14 @@ set(${MODULE_NAME}_SOURCES
     test_workitem.cpp
     test_workgroup.cpp
     test_ifp.cpp
+    test_workgroup_extended_types.cpp
+    workgroup_common_kernels.cpp
+    test_workgroup_non_uniform_vote.cpp
+    test_workgroup_non_uniform_arithmetic.cpp
+    test_workgroup_ballot.cpp
+    test_workgroup_clustered_reduce.cpp
+    test_workgroup_shuffle.cpp
+    test_workgroup_shuffle_relative.cpp
```

Comment:
I find many of the file names (and the corresponding test names) here confusing. There are subgroup functions, not workgroup functions. For some of the existing tests perhaps it made sense as there were equivalent workgroup functions, but for the extended subgroups builtins, this is not the case.

Reply:
fixed